packages/core/cache/src/Cache.js

```diff
@@ -20,17 +19,7 @@ export default class Cache {
     this.dir = cacheDir;
   }
 
-  static deserialize(opts: {cacheDir: FilePath}) {
-    return new Cache(opts.cacheDir);
-  }
-
-  serialize() {
-    return {
-      cacheDir: this.dir
-    };
-  }
-
-  _getCachePath(cacheId: string, extension: string = '.json'): FilePath {
+  _getCachePath(cacheId: string, extension: string = '.v8'): FilePath {
```

Comment:
one potential downside of serializing this way when it comes to the on-disk cache is that it's not as easily introspectable for debugging without some kind of tool, as it has been as json text.

Maybe at some point we could build something that reads these in, deserializes with v8, and displays some kind text representation of it (or maybe something interactable like browser devtools 😃)

Reply:
That's true. I think a tool to inspect the cache might be useful for debugging anyway. Just the file structure of it is somewhat confusing already. Would be better to have a tool that allowed inspecting the cache based on original asset paths or something.